setup.py

```diff
@@ -0,0 +1,10 @@
+from distutils.core import setup
```

Comment:
Right now `python3 setup.py develop` does not work on my system. After looking into it, it seems like the problem can be fixed by switching this line to:
```
from setuptools import setup
```
which works on my local system. Can you try it on your system, and then switch it if it works?

Reply:
Great, thanks for checking. While I was recommending calling setup.py directly I was actually still installing with pip. I carried over use of distutils from other packages I've put together, though see that setuptools is the preferred tool [1].

I've also added the use of `setuptools.find_packages`, else the import would fail under certain conditions, probably because I wasn't using the "packages" line appropriately in setup.py. I've tested installing in a virtual environment with both setup.py directly and with pip and both now work. (A side effect is I added some packages to the `requirements.txt` file.)

[1] - https://packaging.python.org/guides/tool-recommendations/